streams/examples/src/main/java/org/apache/kafka/streams/examples/pageview/PageViewTypedJob.java

```diff
@@ -150,8 +138,7 @@ public PageViewByRegion apply(PageView view, UserProfile profile) {
                         return new KeyValue<>(viewRegion.region, viewRegion);
                     }
                 })
-                .countByKey(HoppingWindows.of("GeoPageViewsWindow").with(7 * 24 * 60 * 60 * 1000),
-                        stringSerializer, stringDeserializer)
+                .countByKey(HoppingWindows.of("GeoPageViewsWindow").with(7 * 24 * 60 * 60 * 1000), SerDes.STRING())
```

Comment:
This line actually shows what we want to achieve:  You only need to pass in `SerDes.STRING()`, yay.